Source/Initialization/InjectorMomentum.H

```diff
@@ -153,34 +153,30 @@ struct InjectorMomentumBoltzmann
     getMomentum (amrex::Real const x, amrex::Real const y, amrex::Real const z,
                  amrex::RandomEngine const& engine) const noexcept
     {
+        using namespace amrex::literals;
+
         // Calculate the local temperature and check if it's too
         // high for Boltzmann or less than zero
         amrex::Real const theta = temperature(x,y,z);
-        if (theta < 0) {
+        if (theta < 0._rt) {
             amrex::Abort("Negative temperature parameter theta encountered, which is not allowed");
         }
         // Calculate local velocity and abort if |beta|>=1
         amrex::Real const beta = velocity(x,y,z);
-        if (beta <= -1 || beta >= 1) {
+        if (beta <= -1_rt || beta >= 1_rt) {
             amrex::Abort("beta = v/c magnitude greater than or equal to 1");
         }
         // Calculate the value of vave from the local temperature
-        amrex::Real const vave = std::sqrt(amrex::Real(2) * theta);
+        amrex::Real const vave = std::sqrt(theta);
```

Comment:
Just double checking: did you remove the `amrex::Real(2)` inside `std::sqrt` in the definition of `vave` because it is included in `amrex::RandomNormal` below?

Reply:
Yes, exactly.

There is usually a factor sqrt(2) in the Box-Mueller transform in order to get a standard (i.e. std of 1) normal distribution. This factor was not present in the old version of the code so the std was `vave`/sqrt(2). To get an equivalent result, we can either remove the factor sqrt(2) in the definition of `vave` (as I did here) or pass `vave`/sqrt(2) instead of `vave` as the second argument to the `amrex::RandomNormal` calls.